sound/shairport-sync/patches/100-mbedtls3fix.patch

```diff
@@ -0,0 +1,95 @@
+--- a/common.c
```

Comment:
This patch is taken from upstream?

Reply:
wrote it in 4.3.3 and brought into upstream as patch

Reply:
Then you should have your patch from https://github.com/mikebrady/shairport-sync/pull/1846.patch , which includes all Git details (Git header, Git author, description), etc. Also, it helps to track the commit, if it was rejected/approved/merged in upstream or not.